handlers/common.go

```diff
@@ -318,8 +318,7 @@ func GetValidatorEarnings(validators []uint64, currency string) (*types.Validato
 	earnings.LastWeekFormatted = utils.FormatIncomeClEl(earnings.Income7d, currency)
 	earnings.LastMonthFormatted = utils.FormatIncomeClEl(earnings.Income31d, currency)
 	earnings.TotalFormatted = utils.FormatIncomeClEl(earnings.IncomeTotal, currency)
-	// earnings.TotalChangeFormatted = utils.FormatIncome(income.ClIncomeTotal+currentDayClIncome+int64(totalDeposits), currency, true)
-	earnings.TotalBalance = utils.FormatElCurrency(totalBalance, currency, 5, true, false, false)
+	earnings.TotalBalance = "<b>" + utils.FormatClCurrency(totalBalance, currency, 5, true, false, false) + "</b>"
```

Comment:
**question:** Should the text be changed to bold?
It is not bold for the balances of the individual validators.

Reply:
The dashboard on mainnet that runs on an older version has this value bold, see for example https://beaconcha.in/dashboard?validators=142321
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/114066135/8ab0c00e-2eaa-4ce8-9e43-20ab27a48e09)

I did not want to change the behavior so I added the <b>.